pkgs/tools/misc/bmon/default.nix

```diff
@@ -12,6 +12,16 @@ stdenv.mkDerivation rec {
     sha256 = "1ilba872c09mnlvylslv4hqv6c9cz36l76q74rr99jvis1dg69gf";
   };
 
+  # https://github.com/Homebrew/homebrew-core/blob/master/Formula/bmon.rb
+  postPatch = ''
+    substituteInPlace include/bmon/bmon.h \
+      --replace "#define __unused__" ""
+    substituteInPlace src/in_proc.c \
+      --replace "__unused__" ""
+    substituteInPlace src/out_curses.c \
+      --replace "__unused__" ""
```

Comment:
IMHO, this looks a bit scary, I would prefer a proper patch instead of "sed".

Apparently this is already reported  upstream:  https://github.com/tgraf/bmon/issues/89. Is upstream dead? Try pinging them.

Reply:
Added a patch.

Reply:
Thanks. Can you also link it to the upstream issue with a comment?

Reply:
Added a comment and a link.

Reply:
Looking at the issue/patch again, why don't you submit this upstream?

Reply:
Simply because upstream seems to be plain dead since 2020.